mission_control/navigator_missions/nav_missions/test.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
Yes, I rebased from dev recently, I'm not sure what happened

GIT SHENANIGANS!